src/robot_interface/models/mission/status.py

```diff
@@ -7,3 +7,9 @@ class StepStatus(str, Enum):
     InProgress: str = "in_progress"
     Failed: str = "failed"
     Cancelled: str = "cancelled"
+
+
+class RobotStatus(Enum):
+    Available: str = "available"
+    Busy: str = "busy"
+    Offline: str = "offline"
```

Comment:
Do we want to add "Deprecated" here aswell as it is comming in Flotilla soon?

Reply:
Maybe, it depends what deprecated means for ISAR. Is the ISAR instance still running? Probably not. 

In any case I think we can wait with adding it until required in this case.

Reply:
Yeah, no I dont think ISAR ever needs to be "Deprecated".